admin-dev/themes/default/template/controllers/orders/helpers/view/view.tpl

```diff
@@ -338,7 +338,7 @@
 										{foreach from=$order->getReturn() item=line}
 										<tr>
 											<td>{$line.date_add}</td>
-											<td>{$line.type}</td>
+											<td>{l s=$line.type}</td>
```

Comment:
you got a point but I can't use translation there cause it's a static method so I can't call the translate method (by $this or even by self:: or static:: )